lib/twine/cli.rb

```diff
@@ -89,6 +89,12 @@ def self.parse(args)
           end
           options[:output_encoding] = e
         end
+        opts.on('-b', '--deliberate', 'Validate the strings file before formatting it') do
```

Comment:
[Agree](http://www.quotesvalley.com/images/02/there-is-only-two-hard-problems-in-computer-science.jpg)